.gitignore

```diff
@@ -1,56 +1,4 @@
-*.gem
-*.rbc
-/.config
 /coverage/
-/InstalledFiles
-/pkg/
-/spec/reports/
-/spec/examples.txt
-/test/tmp/
-/test/version_tmp/
-/tmp/
-
-# Used by dotenv library to load environment variables.
-# .env
-
-## Specific to RubyMotion:
-.dat*
-.repl_history
-build/
-*.bridgesupport
-build-iPhoneOS/
-build-iPhoneSimulator/
-
-## Specific to RubyMotion (use of CocoaPods):
-#
-# We recommend against adding the Pods directory to your .gitignore. However
-# you should judge for yourself, the pros and cons are mentioned at:
-# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
-#
-# vendor/Pods/
-
-## Documentation cache and generated files:
-/.yardoc/
-/_yardoc/
-/doc/
-/rdoc/
-
-## Environment normalization:
-/.bundle/
-/vendor/bundle
-/lib/bundler/man/
-
-# for a library or gem, you might want to ignore these files since the code is
-# intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
-# .ruby-version
-# .ruby-gemset
-
-# unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
-.rvmrc
 
 # Ignore environemnt variables
 .env
-
-# Ignore cassette files
-/test/cassettes/
```

Comment:
Specifically, it's nice if they check in the cassettes so that the tests run cleanly against other Slack workspaces.

Reply:
I observed a lot of students who had problems with accidentally uploading their slack keys because this was left as `/spec/cassettes` when they forked it and students didn't realize they needed to add filtering logic for it (or skipped it because they didn't understand how).
I think we need to either:
- Add the filter logic into the test_helper
- keep this here
- go over adding the filter logic into the test_helper as a whole class activity.

Reply:
I think we should do C.